vendor/knative.dev/pkg/version/version.go

```diff
@@ -33,7 +33,7 @@ const (
 	// NOTE: If you are changing this line, please also update the minimum kubernetes
 	// version listed here:
 	// https://github.com/knative/docs/blob/mkdocs/docs/snippets/prerequisites.md
-	defaultMinimumVersion = "v1.21.0"
+	defaultMinimumVersion = "v1.22.0"
```

Comment:
Looks like the min k8s version after this change will be 1.22. Can we add a release note?

Reply:
yikes, we are actually still on 1.21 for dogfooding

Reply:
Yep, this is why https://github.com/tektoncd/pipeline/pull/4928 was "on hold". 
Do we need absolutely need the version from July 1st or a "released" version (like release-1.5) is ok ? (I know technically it's not really "released" but…) I am asking because we track only released version of all our dependencies *except* knative/pkg 😅 
In any case, we may want to update how we bump knative/pkg (see https://github.com/tektoncd/pipeline/pull/4928/files#diff-ff3402c713d8a504795b7115caa1af088b1bfc583d43adb5d0bc5a83aaa03bc1, it is obsolete).